CrosswordMe.py

```diff
@@ -1,17 +1,46 @@
+from random import randint
+
 #input
 
-words = []
+words = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
 user_input = ""
+print(len(words))
 
-while 1 == 1:
+"""while 1 == 1:
     user_input = input("Word: ")
     if user_input.lower() == "done":
         break
-    words.append(user_input)
+    words.append(user_input)"""
 
 # creating the puzzle
+## horizontals
+line = randint(0,9) #First line to write in
+lines_used = [] #Empty array which contains lines which have been written in already (can probably be removed when its possible to write multiple words in one line)
+
+puzzle = [["$" for x in range(10)] for y in range(10)] #Creates 10x10 array filled with "$"
+
+for word in range(0, len(words)): #loops through every word in the words list
+
+
+
+    word_position = 0 #resets the word_position. this is needed beacuse word_position counts through the length of every word being written
+
+    start_position = randint(0, 10-len(words[word])) #int for start_position
+    end_position = start_position+len(words[word]) #int for end_position
+
+    for array_position in range(start_position, end_position): #Loops through position within the array
+
+        puzzle[line][array_position] = words[word][word_position] #Writes the word at random position of the array
+        word_position += 1 #Fake loop for the word position
 
-puzzle = [[[0 for x in range(10)] for y in range(10)] ]
+    lines_used.append(line) #appends used lines to used lines list
+    line = randint(0,9) #Choosees new line
+    while line in lines_used and word != 9: #keeps creating a new line as long as the line has been used alreardy and its not the last word
+        line = randint(0,9)
 
 
-print(puzzle)
+#Prints the puzzle
+for line in range(10):
+    for position in range(10):
+        print(str(puzzle[line][position]) + " ", end="") #Prints character in current position
+    print() #Prints Newline
```

Comment:
All in all this is a huge waterfall of code.

I'd suggest you look into functions.

Then you could make a function that rotates a word and gives you a matrix of the letters.
You could that checks if a word fits on the board at a specific position.
And a function that writes the word to the board.

And so forth. The more you modularize your code the easier it get's because you don't have to keep all of it in your head at the same time. In the end, you program will likely be:

- Get the list of words to place
- Get or calculate the size of the field
- for every word:
    - find all places
    - at the first possible place
    - if there is no possible place, remove the last word and place that one at it's second possibility
- draw the field once your done.

That's sounds complicated, but if all of these steps are functions, they get really easy to write!